chart/keel/templates/deployment.yaml

```diff
@@ -24,6 +24,7 @@ spec:
 {{ toYaml . | indent 8 }}
 {{- end }}
     spec:
+      securityContext: {{ .Values.securityContext | toYaml | nindent 8 }}
```

Comment:
I am not very familiar with the chart templating engine, but should this not be something like:
```
{{- with .Values.securityContext}}
      securityContext:
{{- toYaml . | nindent 8 }}
{{- end }}
```